textile/fabric_pretreatment/doctype/pretreatment_order/pretreatment_order.py

```diff
@@ -980,7 +980,7 @@ def set_missing_values(source, target):
 
 		if target.meta.has_field("cost_center") and source.get("cost_center"):
 			target.cost_center = source.get("cost_center")
-
+		target.flags.ignore_permissions = ignore_permissions
```

Comment:
Is this necessary? we're already passing ignore permissions to the mapper utility function